functions/manager.py

```diff
@@ -423,32 +423,65 @@ def _runtask(body: Dict[str, Union[str, None]]) -> Boto3Result:
     Keys:
```

Comment:
Add notes about which keys are required? Some of the other functions include a `Required.` at the end of a key's description.